### DISCUSSION TITLE
PHP Fatal error:  Uncaught Error: Class "Tectalic\OpenAi\Manager" not found

### DISCUSSION THREAD

**Discussion by wyatt121** [tectalichq/public-openai-client-php#10] [Q&A] [answered]
Hi,

When I run the example PHP code from your website with my API key, I get the following error:

PHP Fatal error:  Uncaught Error: Class "Tectalic\OpenAi\Manager" not found in /var/www/html/scrape/chatgpt/test.php:3
Stack trace:
#0 {main}
  thrown in /var/www/html/scrape/chatgpt/test.php on line 3

What am I doing wrong?  Any help is much appreciated.

Wyatt

**Comment by thejamescollins** [comment, marked as answer]
Hi there @wyatt121,

Does your `test.php` script require/include the Composer autoloader file?

Please see the [Usage section here](https://tectalic.com/apis/openai/docs/php#installation).

1. Install the package: `composer require tectalic/openai`
2. Load your project's composer autoloader (if you aren't already doing so):
`require_once(__DIR__ . '/vendor/autoload.php');`
3. Then build the Tectalic Client.

If that doesn't help, please copy and paste your `test.php` file and include it here so we can see the code you're using.

Thank you,

James
